bin/knit-frog.py

```diff
@@ -109,6 +109,10 @@
 cmdhandle.write('echo "$(hostname -f) ansible_connection=local ansible_user=$(whoami)"  >> ~/.hosts\n')
 cmdhandle.write("echo '[pulsarservers]' >> ~/.hosts\n")
 cmdhandle.write('echo "$(hostname -f) ansible_connection=local ansible_user=$(whoami)"  >> ~/.hosts\n')
+cmdhandle.write("echo '[dbservers]' >> ~/.hosts\n")
```

Comment:
this should be `dbservers:children`, to match what is in ansible-galaxy, right?

Reply:
I am sorry I am unable to associate which one you are comparing it to. This [file](https://github.com/hexylena/git-gat/blob/2023-04-05/galaxy.yml#L2) uses the hosts group `dbservers`.

Reply:
https://github.com/hexylena/git-gat/blob/2023-04-05/hosts#L3

Reply:
Sure! I will change it to `parent:children` and then the hostname line will be replaced by the `galaxyservers` group. I will push a commit.